components/ToDoList.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <v-container>
+        <v-card>
+            <h1 
+                :style="{
+                    color: inputValue.length < 5 ?
```

Comment:
please move logic to computed value

Reply:
Please, could you show me how to dot it? Thank you.

Reply:
yes here is an example: 
```
computed: {
    h1Color () {
        return this.inputValue.length < 5 ? 'darkred' : 'green';
    }
}
```

and then in your template `style="{ color: h1Color }"`